### PR TITLE
chore: fix OS3 integration tests

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [X] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Fix OS3 integration tests

### CODE REVIEW

File: test/integration/kubernetes.spec.ts [hunk 1]
```diff
@@ -90,7 +101,11 @@ test('deploy sample workloads', async () => {
           argoNamespace,
         ),
       )
-      .then(() => kubectl.applyK8sYaml('./test/fixtures/argo-rollout.yaml')),
+      .then(() => kubectl.applyK8sYaml('./test/fixtures/argo-rollout.yaml'))
+      .catch((error) => {
```
Comment (kat1906) [inline test/integration/kubernetes.spec.ts:105]: this error could come from anywhere in the chain?

Comment (kat1906) [reply test/integration/kubernetes.spec.ts:105]: so might not be argoRollout

Comment (dkontorovskyy) [reply test/integration/kubernetes.spec.ts:105]: No, it's only attached to `argo` flow, so irrelevant to other flows

Comment (kat1906) [reply test/integration/kubernetes.spec.ts:105]: oh I see, this is all argo chain

**Review by kat1906 — APPROVED** [review]

**Pull request merged**